Code/Legacy/CrySystem/SystemInit.cpp

```diff
@@ -896,16 +896,14 @@ void CSystem::InitLocalization()
     if (auto console = AZ::Interface<AZ::IConsole>::Get(); console != nullptr)
     {
         AZ::CVarFixedString languageAudio;
-        if (auto result = console->GetCvarValue("g_languageAudio", languageAudio); result == AZ::GetValueResult::Success)
+        console->GetCvarValue("g_languageAudio", languageAudio);
+        if (languageAudio.size() == 0)
```

Comment:
nit - if (string.empty()) is preferred when checking for is empty...

Reply:
Will update, thanks.